src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Event;
+import com.google.sps.TripDay;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/get-calendar")
+public class CalendarServlet extends HttpServlet {
```

Comment:
This servlet seems to be just pulling TripDays, why is it called CalendarServlet? I feel like that could be useful for other parts of the system other than Calendar?

Reply:
I don't know how Eshika is planning on puling information into the maps frontend but she doesn't use Events. I think she is planning on using a DirectionRequest object or something like that but if she can do that in this servlet as well than we can do that as well! The only problem is that in order to communicate with my calendar I need the events to be the only thing written by the `response.writer()`

Reply:
Have you discussed with Eshika and Chris what we talked about this morning about having utility classes to do these repetitive calls for you?